### PR TITLE
Added !paste

### PR DESCRIPTION
**Pull request by TheOnlyMaldred**
Added pastebin quick link (freely change to Hastebin if necessary)

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -163,6 +163,11 @@
                     "value": "https://ci.lucko.me/view/LuckPerms/"
                 }
             ]
+        },
+            "command": "paste",
```
Comment (Andre601) [inline data.json]: Here is a { missing
```suggestion
        {
            "command": "paste",
```

File: data.json [hunk 2]
```diff
@@ -163,6 +163,11 @@
...
+            "title": "PasteBin",
```
Comment (Andre601) [inline data.json]: "PasteBin" looks stupid in terms of capitalization.
Also, hastebin is preferred over Pastebin as of the reasons BrainStone already mentioned on Discord.
```suggestion
            "title": "Hastebin",
```

File: data.json [hunk 3]
```diff
@@ -163,6 +163,11 @@
...
+            "url": "https://pastebin.com/",
+            "description": "We don't like seeing chat flooded with error codes... Paste your error codes or configuration files in here and share the link!",
```
Comment (Andre601) [inline data.json]: That spelling sounds a bit annoyed/angry.
It's not like we don't like people sharing errors through Discord, but that hastebin, Pastebin, etc. are a better alternative to more easily read the text.

Also, you can add alternative links to the response.
```suggestion
            "description": "Hastebin allows to easily share config files or errors without making a mess.\n**Please make sure to remove passwords, usernames, etc. when sharing config files or similar!**",
            "fields": [
                {
                    "key": "Pastebin",
                    "value": "https://pastebin.com/"
                },
                {
                    "key": "Paste.helpch.at",
                    "value": "https://paste.helpch.at"
                }
            ]
```

File: data.json [hunk 4]
```diff
@@ -163,6 +163,11 @@
                     "value": "https://ci.lucko.me/view/LuckPerms/"
                 }
             ]
+        },
+            "command": "paste",
+            "title": "PasteBin",
+            "url": "https://pastebin.com/",
```
Comment (Andre601) [inline data.json]: Hastebin > Pastebin
```suggestion
            "url": "https://hasteb.in/",
```

File: data.json [hunk 5]
```diff
@@ -163,6 +163,11 @@
                     "value": "https://ci.lucko.me/view/LuckPerms/"
                 }
             ]
+        },
```
Comment (TheOnlyMaldred) [inline data.json]: Missed an open curly bracket here

**Comment by Turbotailz** [comment]
The only issue I see with adding things that aren't in the wiki is that the first embed field will always say "Read more" before the URL and the following fields have configurable titles. I might make a change that lets us define commands as "external" so that we can edit the first field title in this case.

**Comment by Andre601** [comment]
That would indeed be a good change.
Perhaps just make it, that the "Read more" field is just yet another in the fields array.
Would imo be the best option to have.

**Comment by Andre601** [comment]
I think this command is no longer needed, since there is now a command for paste-sites (Lucko's bytebin f.e.)

**Comment by lucko** [comment]
Yeah, sorry this got merged in with another PR - thank you though!

**Pull request closed without merging**